modules/dns/variables.tf

```diff
@@ -49,3 +49,8 @@ variable "dns_record_set" {
     error_message = "The specified DNS record type is not valid."
   }
 }
+
+variable "dns_records_file" {
+  type        = string
+  description = "The DNS zone file that contains the details of the DNS records."
```

Comment:
Good to have a link of the file format as an example.

Reply:
Added the link.